src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  & :not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--active :not(:last-child) {
+    background-image: url(/src/images/star-active.svg);
```

Comment:
Ensure that the image `star-active.svg` exists at the path `/src/images/`. If it's not present, the active star background will not display correctly.